### ISSUE TITLE
Hotkey for each device

### ISSUE THREAD

**Issue by snebjorn** [sirWest/AudioSwitch#40]
I like to be able to bind a key that switches to my headset (that includes a mic). And another to my speakers.

**Comment by rudydevolder** [comment]
Yes, this would be awesome! Because I have a blind friend and so he can switch easely to his external digital speakersystem!

**Comment by VVelda** [comment]
+1 That would be awesome. For me, switch audio device key shortcut is ideal. :)

**Comment by dmitrijs** [comment]
+1

**Comment by EpicLPer** [comment]
This is basically the reason why I've downloaded this so I can use hotkeys to cycle through the 3 main outputs I have on my PC. But I figured it doesn't do that yet...
